### ISSUE TITLE
Requirements are a little strict? And IUPAC name of O=Cc1ccccc1 is: styrene 

### ISSUE THREAD

**Issue by willfinnigan** [Kohulan/Smiles-TO-iUpac-Translator#9]
Hi,

Awesome package.  Just a comment that I wanted to install stout into an existing environment I'm using for Chemoinformatics.  This environment has rdkit installed through conda, and tensorflow 2.6.2 (not tensorflow-gpu).  I installed pystow manually and then ignored the requirements for stout, and it works fine.  Just wondering if the requirement for tensorflow-gpu and the pypi version of rdkit is a bit strict?

So anyway all working nicely.  I ran a bunch of test molecules, which mostly look great.  
However this one caught my eye. 
IUPAC name of O=Cc1ccccc1 is: styrene.    Should be benzaldehyde.  
Just thought I'd flag this as a test case, as its a relatively simple molecule.  Not meant as a critisism or anything.

Keep up the great work.

Kind regards,
Will

**Comment by Kohulan** [comment]
Hi Will,

Thanks a lot for your message. We tested STOUT those particular versions that's we made those versions hardcoded so it won't break anything for the end user.

"IUPAC name of O=Cc1ccccc1 is styrene. " this produces the name "Benzaldehyde" because we trained the model using PubChem IUPAC names and kekulized SMILES so every molecule you use as input will get kekulized internally. 
https://pubchem.ncbi.nlm.nih.gov/compound/240

Hope this helps. 😄 

Kind regards,
Kohulan
